index.js

```diff
@@ -149,42 +192,68 @@ const line = []
 
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
 
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 1`
+counter = 0
+
+function takeATicketNumber(){
+  counter += 1
+  console.log(`Welcome. You are ticket number ${counter}`)
+  line.push(counter)
+}
```

Comment:
This works, but make sure to declare variables using `let` or `const` - on line 197:

```
let counter = 0
```